mithril-common/src/cardano_transactions_preloader.rs

```diff
@@ -131,11 +185,57 @@ mod tests {
             security_parameter,
             Arc::new(chain_observer),
             TestLogger::stdout(),
+            Arc::new(CardanoTransactionsPreloaderActivation::new(true)),
+        );
+
+        preloader.preload().await.unwrap();
+    }
+
+    #[tokio::test]
+    async fn do_not_call_its_inner_importer_when_is_not_activated() {
+        let security_parameter = 542;
```

Comment:
This variable could be inlined here and in tests below.

Reply:
Done ✅